openapi_tester/exceptions.py

```diff
@@ -65,15 +73,15 @@ def __init__(self, key: str, case: str, expected: str) -> None:
         super().__init__(f"The response key `{key}` is not properly {case}. Expected value: {expected}")
 
 
-class OpenAPISchemaError(Exception):
+class LoadingError(Exception):
```

Comment:
this name is worse then the previous name - LoadingError is too generic. OpenAPISchemaError is a better name.